packages/api-sdk/src/shared.ts

```diff
@@ -20,6 +20,6 @@ export const ISO_DATE = "YYYY-MM-DD";
 export const defaultString = z.string().trim();
 export const defaultOptionalString = optionalString(defaultString);
 export const defaultDateString = defaultString.refine(v => dayjs(v, ISO_DATE, true).isValid(), {
-  message: `Date must be a valid ISO 8601 date formatted ${ISO_DATE}. Example: 2023-05-03`,
+  message: `Date must be a valid ISO 8601 date formatted ${ISO_DATE}. Example: 2023-15-03`,
```

Comment:
shouldn't this be the other way around since it's YYYY-MM-DD?

ie `2023-03-15` or something

Reply:
good catch! 🕵️